Update documentation (#235)
xCDAT/xcdat: docs/faqs.rst

Frequently Asked Questions
==========================

Data Wrangling
--------------

> Some datasets might have quality issues, such as non-CF compliant attributes and inconsistent values.

Author: ```suggestion
Some datasets might have data quality issues such as inconsistent floating point values between files or non-CF compliant attributes.
```

> As a result, ``xarray`` and ``xcdat`` not being able to open these datasets using the default settings.

Author: ```suggestion
If data quality issues are present, `xarray`` and ``xcdat`` might not be able to open the datasets.

You might be able to workaround data quality issues by:
1. Configuring ``open_dataset()`` or ``open_mfdataset()`` keyword arguments based on your needs.
2. Writing a custom `preprocess()` function to feed into ``open_mfdataset()``, which preprocesses each dataset file individually before joining them into a single Dataset object.

``xcdat`` aims to be generalized library, so we don't intend on implementing functionality to fix poor data for limited use cases.
```

Frequently Asked Questions
==========================

Data Wrangling
--------------

``xcdat`` aims to implement generalized functionality. This means that functionality intended to handle data quality issues is out of scope, especially for limited cases.

If data quality issues are present, ``xarray`` and ``xcdat`` might not be able to open the datasets.
Examples of data quality issues include conflicting floating point values between files or non-CF compliant attributes.

A few workarounds include:

1. Configuring ``open_dataset()`` or ``open_mfdataset()`` keyword arguments based on your needs.
2. Writing a custom ``preprocess()`` function to feed into ``open_mfdataset()``. This function preprocesses each dataset file individually before joining them into a single Dataset object.

> 

Author: ```suggestion
```
